specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,103 @@
+| Developer Tools           | Other (Developer Tools)           | All Developer Tools services.                                                                                                                                                                          |
+| Identity                  | Other (Identity)                  | All Identity services.                                                                                                                                                                                 |
+| Integration               | API Management                    | Creation, publishing, and governance of application programming interfaces that facilitate communication between software systems.                                                                     |
+|                           | Messaging                         | Asynchronous communication between distributed applications.                                                                                                                                           |
+|                           | Workflow Automation               | Use of software to complete orchestration tasks and activities without the need for human input.                                                                                                       |
+|                           | Other (Integration)               | Integration services that do not fall into one of the defined subcategories.                                                                                                                           |
+| Internet of Things        | Other (Internet of Things)        | All Internet of Things (IoT) services.                                                                                                                                                                 |
+| Management and Governance | Administration                    | Management and maintenance of cloud systems.                                                                                                                                                           |
+|                           | Architecture                      | Planning, design, and construction of cloud systems.                                                                                                                                                   |
+|                           | Compliance                        | Adherance to regulatory standards and industry best practices.                                                                                                                                         |
+|                           | Cost Management                   | Monitoring and controlling expenses of cloud systems.                                                                                                                                                  |
```

Comment:
```suggestion
|                           | Cost Management                   | Monitoring and controlling expenses of systems and services.                                                                                                                                                  |
```

Reply:
Is `cloud` a verboten term in the spec?  Based on your comments, I assume `systems and services` is preferred?  It would be helpful to add that to the editorial guidelines, or maybe it is and I didn't notice.